From d3a62fdbabba4cc98cf7b412a7dbc315df8e891f Mon Sep 17 00:00:00 2001
From: "lijun.xie" <lijun.xie@ck-telecom.com>
Date: Tue, 21 Jul 2015 00:50:52 -0700
Subject: [PATCH 35/74] FM: The collect channel disappear when quit FM

[Procedures]:
    1.FM->collect one channel
    2.Quit FM and enter FM
    3.Check the collect channel

[Expect results]:
The collect channel exist

[Actual results]:
The collect channel disappear

Change-Id: I50f9f8d849d7d1bfab1159be51e9ad0ef2bac0a9
---
 fmapp2/src/com/caf/fmradio/FMRadio.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fmapp2/src/com/caf/fmradio/FMRadio.java b/fmapp2/src/com/caf/fmradio/FMRadio.java
index 28a91cd..0cc8e71 100644
--- a/fmapp2/src/com/caf/fmradio/FMRadio.java
+++ b/fmapp2/src/com/caf/fmradio/FMRadio.java
@@ -1932,6 +1932,7 @@ public class FMRadio extends Activity
          FmSharedPreferences.addStation(selectedStation.getName(), selectedStation
                         .getFrequency(), currentList);
          setupPresetLayout();
+         mPrefs.Save();
       }
    }
 
-- 
2.17.1

